tgui/packages/tgui/interfaces/PaiInterface/constants/index.ts

```diff
@@ -0,0 +1,61 @@
+export const DIRECTIVE_COMPREHENSION: string = `As an advanced software
```

Comment:
You don't need a new folder for this file. Just call it constants.ts and import it in from index.tsx
A folder like this is only useful if you plan to have the constants separated into different files